libbeat/instrumentation/instrumentation.go

```diff
@@ -25,17 +25,16 @@ import (
 	"os"
 	"time"
 
-	"go.elastic.co/apm"
-	apmtransport "go.elastic.co/apm/transport"
+	"go.elastic.co/apm/v2"
+	apmtransport "go.elastic.co/apm/v2/transport"
 
 	"github.com/elastic/beats/v7/libbeat/common"
 	"github.com/elastic/beats/v7/libbeat/common/transport"
 	"github.com/elastic/beats/v7/libbeat/logp"
 )
 
 func init() {
-	// we need to close the default tracer to prevent the beat sending events to localhost:8200
```

Comment:
Was the comment here a lie?

Reply:
We _should_ remove this code.

It used to be that importing `go.elastic.co/apm` would create a default tracer that would actively try to connect to localhost:8200, trying to fetch central config and periodically send metrics. With v2, we now defer creation of this default tracer until `apm.DefaultTracer()` is called.

Reply:
sounds good, I will remove the DefaultTracer calls.